weblate/addons/webhooks.py

```diff
@@ -0,0 +1,76 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+from django.utils import timezone
+from django.utils.translation import gettext_lazy
+
+from weblate.addons.base import ChangeBaseAddon
+from weblate.addons.forms import WebhooksAddonForm
+from weblate.api.tasks import webhook_delivery_task
+from weblate.trans.util import split_plural
+
+
+class WebhookAddon(ChangeBaseAddon):
+    name = "weblate.webhook.webhook"
+    # TODO: improve description
+    verbose = gettext_lazy("Webhooks")
+    description = gettext_lazy("some desc")
+
+    settings_form = WebhooksAddonForm
+    # TODO: find a webhook icon
```

Comment:
We use MDI, there is webhook icon: https://pictogrammers.com/library/mdi/icon/webhook/, you need to download SVG and include it in static files.